CaPPMS/Shared/ProjectEntryAuthorizedDropdown.razor

```diff
@@ -335,27 +364,36 @@
 
     private async Task ApproveIdea(MouseEventArgs e)
     {
-        var gitHubOrganization = "umgc";
-        var formattedRepoName = projectInformation.ProjectTitle.Replace(' ', '-');
-        try
-        {
-            this.gitHubState = "none";
-            await GitHub.DoAllTasks(gitHubOrganization, formattedRepoName, projectInformation.ProjectDescription).ConfigureAwait(false);
-            this.gitHubState = "block";
-        }
-        catch (Exception ex)
+        if (createGitHubRepository)
         {
-            Console.WriteLine("Error Message - {0}", ex.Message);
-            this.gitHubState = "block";
-            await Task.Delay(2000).ConfigureAwait(false);
-            this.gitHubErrorState = "none";
-            return;
+            var gitHubOrganization = "umgc";
+            var formattedRepoName = this.gitHubRepositoryName.Replace(' ', '-').ToLower();
+            try
+            {
+                this.gitHubState = "none";
+                gitHubError = await GitHub.DoAllTasks(gitHubOrganization, formattedRepoName, projectInformation.ProjectDescription);
```

Comment:
Get the proper error out of the GitHub.DoAllTasks method and give it to the user. At this point, the user is expected to be one of the admins.